diff --git a/apc/PowerNet-MIB b/apc/PowerNet-MIB
index af0da3b..2dd530e 100644
--- a/apc/PowerNet-MIB
+++ b/apc/PowerNet-MIB
@@ -399,7 +399,7 @@ PowerNet-MIB DEFINITIONS ::= BEGIN
 
 IMPORTS
    enterprises, IpAddress, Gauge, TimeTicks             FROM RFC1155-SMI
-   DisplayString                                        FROM RFC1213-MIB
+   DisplayString                                        FROM SNMPv2-TC
    SnmpAdminString                                      FROM SNMP-FRAMEWORK-MIB
    OBJECT-TYPE                                          FROM RFC-1212
    TRAP-TYPE                                            FROM RFC-1215;
